core/ui/PageTemplate/sidebar-resizer.tid

```diff
@@ -0,0 +1,36 @@
+title: $:/core/ui/PageTemplate/sidebar-resizer
+tags: $:/tags/PageTemplate
+list-after: $:/core/ui/PageTemplate/story
+code-body: yes
+
+\import [function[get.base.functions.theme],<get.current.theme>first[]is[tiddler]] :else[function[get.base.functions.theme],<get.current.theme>first[]is[shadow]] :else[[$:/themes/tiddlywiki/vanilla/functions]] [[$:/core/procedures/sidebar-resizer]]
+\function resizer.state.tiddler() $:/state/sidebar/resizing
+\function resizer.state() [{$:/state/sidebar}]
+\function resizer.class() tc-main-sidebar-resizer
+\function set.throttling() yes
+\function drag.direction.reverse() [<get.theme.option sidebarposition>match[left]then[yes]] :else[[no]]
+\whitespace trim
+
+<%if [<get.theme.explicit.option sidebarresizer>match[show]] %>
+
+	<$let
+		tv-set-storywidth-storyright=<<set.storywidth.storyright>>
+		tv-set-sidebarwidth=<<set.sidebarwidth>>
+		tv-set-centralised=<<set.centralised>>
+
+		storyLeftTiddler=<<get.story-left.tiddler>>
+		storyRightTiddler=<<get.story-right.tiddler>>
+		sidebarWidthTiddler=<<get.sidebar-width.tiddler>>
+		tiddlerWidthTiddler=<<get.tiddler-width.tiddler>>
+		storyWidthTiddler=<<get.story-width.tiddler>>
+		storyMinWidthTiddler=<<get.story-minwidth.tiddler>>
+		sidebarMinWidthTiddler=<<get.sidebar-minwidth.tiddler>>
+		storyPaddingLeftTiddler=<<get.story-padding-left.tiddler>>
+		storyPaddingRightTiddler=<<get.story-padding-right.tiddler>>
+	>
+
+		<$transclude $variable="sidebar-resizer"/>
```

Comment:
IMO the leading and trailing new-lines should be removed. I am not sure if `<$transclude` will need a `$mode="block"` or not. But there should be no empty lines here